app/models/point.rb

```diff
@@ -0,0 +1,4 @@
+class Point < ApplicationRecord
```

Comment:
It's just a Rails convention thing. Here's a good explanation: 

```Ruby on Rails follow linguistic convention. That means a model represents a single user, whereas a database table consists of many users.```

http://stackoverflow.com/questions/10078139/ruby-on-rails-plural-controller-and-singular-model-convention-explanation